snowshu/adapters/source_adapters/snowflake_adapter.py

```diff
@@ -103,6 +103,100 @@ def _get_all_schemas(self, database: str, exclude_defaults: Optional[bool] = Fal
             f'Done. Found {len(schemas)} schemas in {database} database.')
         return schemas
 
+    def _get_all_tables(self, database: str, schema: str) -> List[str]:
+        database = self.quoted(database)
+        schema = self.quoted(schema)
+        logger.debug(f'Collecting tables from {schema} schema in {database} database in snowflake...')
+        show_result = self._safe_query(f'SHOW TERSE TABLES IN SCHEMA {database}.{schema}')['name'].tolist()
+        tables = list(set(show_result))
```

Comment:
Are there any cases when SHOW TERSE TABLES IN SCHEMA return duplicate names?

Reply:
Normally I'd say not, as Snowflake forces name uniqueness, though I found one bypass, it's using quotes to create tables eg:

```
CREATE TABLE "myTable" (id INT);
CREATE TABLE "MYTABLE" (id INT);  -- This will succeed
```

Do you see a scenario where it can com back to bite us?